test/examples/30_pybind11_examples/02_advanced/03_exceptions/exceptions.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+import exceptions_pybind11 as py11
+import exceptions_boost_python as boost
```

Comment:
What does boost_python do here?

Reply:
We test both pybind11 and boost_python. See [lines 21-22](https://github.com/personalrobotics/chimera/pull/290/files#diff-7f929e495012676319cf582f7f0987f6R21-R22) in this file.